packages/nimble-components/src/unit/celsius/tests/unit-celcius.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { UnitCelsius, unitCelsiusTag } from '..';
```

Comment:
File name has a typo: `celcius` -> `celsius`

Reply:
Ugh. My brain really wants to spell it wrong.